android/java/org/chromium/chrome/browser/BraveRewardsVerifyWalletActivity.java

```diff
@@ -63,7 +97,8 @@ void SetUpholdLinkHandler() {
 
                         if (BraveRewardsHelper.subtextAtOffset(built_service_str, part2, offset) ){
                             Intent intent = new Intent();
-                            intent.putExtra(BraveActivity.OPEN_URL, BraveUphold.UPHOLD_ORIGIN_URL);
+                            intent.putExtra(
+                                    BraveActivity.OPEN_URL, BraveWalletProvider.UPHOLD_ORIGIN_URL);
```

Comment:
@deeppandya If this function is Uphold-specific, do we need a bitFlyer-specific function. If it's not Uphold-specific, then it seems like we need to be smarter here (and probably should rename the function, etc.)